rules-java-archives/addon/pom.xml

```diff
@@ -58,10 +62,50 @@
             <classifier>forge-addon</classifier>
             <scope>provided</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>${version.nexus.index}</version>
+            <type>zip</type>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
```

Comment:
IMO, resources are files that are maintained by other team than development - e.g. icons, translated text files, properties with configuration, etc. which was also the reason for splitting the directories. beans.xml is a source. It is only impractical burden to keep them somewhere else. But I succumb :)